server/app/routes/v1/services/service_container_logs.rb

```diff
@@ -5,7 +5,7 @@
     r.is do
       scope = @grid_service.container_logs
 
-      scope = scope.where(name: r['container']) unless r['container'].nil?
```

Comment:
Dunno, just include both `where`'s in the scope... I can't think of any usecase for that. The `?container` would just be for backwards-compatibility.